tests/impl.cpp

```diff
@@ -3117,7 +3117,17 @@ result_t test_mm_blendv_epi8(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int32_t *_a = (const int32_t *) impl.mTestIntPointer1;
+    const int count = 3;
```

Comment:
Here `count` is `imm8`. You can do another test for the condition `imm8 > 31`.

Reply:
Can this be set to random value?

Reply:
Any progress?

Reply:
I choose to use fix value

Reply:
But the other situations would not be tested?

Reply:
Or let me test all the cases

Reply:
I think you can use the pass-in argument `i` and do the modulo calculation on it.
Like `int count = i % x`, x can be a value which is larger than 32.